integrated_channels/degreed2/models.py

```diff
@@ -178,6 +178,11 @@ class Degreed2LearnerDataTransmissionAudit(models.Model):
         help_text="Used as content-id field when creating a completion"
     )
 
+    course_completed = models.BooleanField(
```

Comment:
Cribbed from the Degreed1 version. Required by this code:

https://github.com/edx/edx-enterprise/blob/jnagro/ENT-5558/1/integrated_channels/integrated_channel/transmitters/learner_data.py#L267

Another example where a standard parent class for LearnerDataTransmissionAudit models would help us.